### PR TITLE
network(engine): return network usage by GetJob

### LINKED ISSUE #7801: engine(dm): get network usage by GetJobByID
Used for dm-on-cloud

### PR DESCRIPTION
**Pull request by buchuitoudegou**
### What problem does this PR solve?

Issue Number: close #7801 

### What is changed and how it works?
Add IOTotalBytes to task config.

But the network usage from Dumpling and Lightning cannot be fetched in this way.

#### Tests 

 - Unit test
 - Integration test

### Release note 

```release-note
`None`.
```

### CODE REVIEW

**Review by lance6716 — APPROVED** [review]

File: dm/syncer/dbconn/db.go [hunk 1]
```diff
@@ -268,6 +268,7 @@ func (conn *DBConn) retryableFn(tctx *tcontext.Context, queries, args any) func(
 // CreateConns returns a opened DB from dbCfg and number of `count` connections of that DB.
 func CreateConns(tctx *tcontext.Context, cfg *config.SubTaskConfig, dbCfg *config.DBConfig, count int) (*conn.BaseDB, []*DBConn, error) {
 	if cfg.IOTotalBytes != nil {
+		tctx.L().Info("create connection with io total bytes")
```
[the remark this answers is not in the document]
Comment (buchuitoudegou) [reply dm/syncer/dbconn/db.go:271]: 👌

**Comment by maxshuang** [comment]
Rest LGTM.

File: engine/executor/dm/api.go [hunk 2]
```diff
@@ -42,10 +42,11 @@ func (w *dmWorker) QueryStatus(ctx context.Context, req *dmpkg.QueryStatusReques
...
 	return &dmpkg.QueryStatusResponse{
-		Unit:   w.workerType,
-		Stage:  stage,
-		Result: dmpkg.NewProcessResultFromPB(result),
-		Status: buf.Bytes(),
+		Unit:         w.workerType,
+		Stage:        stage,
+		Result:       dmpkg.NewProcessResultFromPB(result),
+		Status:       buf.Bytes(),
+		IoTotalBytes: w.cfg.IOTotalBytes.Load(),
```
[the remark this answers is not in the document]
Comment (buchuitoudegou) [reply engine/executor/dm/api.go:49]: https://github.com/pingcap/tiflow/pull/7538

**Review by maxshuang — APPROVED** [review]

**Pull request merged**